app/src/main/java/com/waseefakhtar/doseapp/feature/addmedication/AddMedicationRoute.kt

```diff
@@ -98,280 +102,288 @@ fun AddMedicationScreen(
     var isAfternoonSelected by rememberSaveable { mutableStateOf(false) }
     var isEveningSelected by rememberSaveable { mutableStateOf(false) }
     var isNightSelected by rememberSaveable { mutableStateOf(false) }
+    val context = LocalContext.current
 
-    Column(
-        modifier = Modifier
-            .padding(0.dp, 16.dp)
-            .verticalScroll(rememberScrollState()),
-        verticalArrangement = Arrangement.spacedBy(8.dp)
-    ) {
-        Row(
-            horizontalArrangement = Arrangement.spacedBy(16.dp),
-            verticalAlignment = Alignment.CenterVertically
-        ) {
-            FloatingActionButton(
+    Scaffold(
```

Comment:
Will the form be scrollable on smaller screens?

Reply:
Yes, it is scrollable